app/helpers/reports_helper.rb

```diff
@@ -3,90 +3,24 @@
 module ReportsHelper
   include StringUtility
 
-  def render_new_element(hide)
-    render partial: 'reports/elements/new_element.html.erb',
-          locals: { hide: hide }
-  end
-
   def render_report_element(element, provided_locals = nil)
     # Determine partial
-    file_name = element.type_of
-    if element.type_of.in? ReportExtends::MY_MODULE_CHILDREN_ELEMENTS
-      file_name = "my_module_#{element.type_of.singularize}"
-    end
-    view = "reports/elements/#{file_name}_element.html.erb"
+    view = "reports/elements/#{element.type_of}_element.html.erb"
 
     # Set locals
-
     locals = provided_locals.nil? ? {} : provided_locals.clone
 
     children_html = ''.html_safe
     # First, recursively render element's children
-    if element.comments? || element.project_header?
-      # Render no children
-    elsif element.result?
-      # Special handling for result comments
-      if element.children.active.present?
-        children_html.safe_concat render_new_element(true)
-        element.children.active.each do |child|
-          children_html
-            .safe_concat render_report_element(child, provided_locals)
-        end
-      else
-        children_html.safe_concat render_new_element(false)
+    if element.children.active.present?
+      element.children.active.each do |child|
+        children_html.safe_concat render_report_element(child, provided_locals)
```

Comment:
`element.children.active&.each do |child|`
would do the same without the if block